phi/phi-3.5-mini/config.yaml

```diff
@@ -0,0 +1,18 @@
+model_name: "Phi 3.5 Mini Instruct VLLM openai compatible"
+python_version: py311
+model_metadata:
+  example_model_input: {"prompt": "what is the meaning of life"}
+  repo_id: microsoft/Phi-3.5-mini-instruct
+  openai_compatible: true
+  vllm_config:
+    tensor_parallel_size: 1
+    max_model_len: 10000
+requirements:
+  - vllm==0.5.4
+resources:
+  accelerator: A100
```

Comment:
let's use a smaller GPU, since the beauty of a smaller model is that we can do that!